### ISSUE TITLE
[Update] API v1.1.0 doc

### ISSUE THREAD

**Issue by godmmt** [CAFECA-IO/BAIFA#551]
- [x] 移除 `addressId`
013 - Get red flags data from an Address
019 - Get red flags data from a currency
021 - Get all red flags data / Get red flags data from a currency, chain, or address

- [x] 移除 `addressId`、`address`
022 - Get data from a red flag

_Originally posted by @Julian0701 in https://github.com/CAFECA-IO/BAIFA/issues/495#issuecomment-1915926994_

---

### 文件位置：[API v1.1.0](https://github.com/CAFECA-IO/BAIFA/wiki/API-v1.1.0)

---

- [x] 021 - 刪除一支API
HTTP Request: `GET /app/red_flags?currency=${currencyId}&chain=${chainId}&address=${addressId}`
- [x] 移除所有 `chainIcon`
影響範圍：004 / 005 / 007 / 010 / 011 / 012 / 013 / 014 / 018 / 022 
- [x] 003 - 移除 `address`並且新增:
  ```json
  "interactedAddresses": [
    {
      "id": "122134",
      "chainId": "eth"
    },
    {
      "id": "134325",
      "chainId": "usdt"
    }
  ],
  ```

**Comment by Julian0701** [comment]
移除所有 `chainIcon`

**Comment by godmmt** [comment]
API v1.1.0 Version : [d5d14b4](https://github.com/CAFECA-IO/BAIFA/wiki/API-v1.1.0/d5d14b45c990392ba6939ca801f429e616b8fd7c)

**Comment by godmmt** [comment]
took 1 hour

done.

**Comment by godmmt** [comment]
003 - 移除 address並且新增:
```json
"interactedAddresses": [
  {
    "id": "122134",
    "chainId": "eth"
  },
  {
    "id": "134325",
    "chainId": "usdt"
  }
],
```

![截圖 2024-01-30 上午11 45 05](https://github.com/CAFECA-IO/BAIFA/assets/105651918/bd53a381-db4a-4aab-ac73-77f9ebecda28)

**Comment by godmmt** [comment]
took 1 hour

done.

**Comment by godmmt** [comment]
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/401
      API adjustment : 005 / 006 / 009 #401
      --8
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/356
      [MainTask] 機器人聊天室 #356
      --1
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/412
      Backend Development #412
      --8
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/53
      上傳 KM(Git Flow) 到 MerMer website #53
      --5
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/54
      [MainTask] KM: 撰寫《新手指南》 #54
      --1
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/66
      [SubTask]docsCreation : 1. 團隊例會與準備 #66
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/67
      [SubTask]docsCreation : 2. 開立工作票 #67
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/68
      [SubTask]docsCreation : 3. 需求規格書撰寫 #68
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/72
      [SubTask]docsCreation : 7. 專案建立 #72
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/73
      [SubTask]docsCreation : 8. 程式碼撰寫 #73
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/74
      [SubTask]docsCreation : 9. 版本管理 #74
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/75
      [SubTask]docsCreation : 10. 品質管理 #75
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/76
      [SubTask]docsCreation : 11. 版本釋出 #76
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/77
      [SubTask]docsCreation : 12. KM 撰寫 #77
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/79
      [SubTask]整合所有子任務的文章＆校稿＆存放 #79
      --3
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/423
      [Improvement] send icon, text width, delete useless data #423
      --8
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/401
      API adjustment : 005 / 006 / 009 #401
      --8
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/424
      API 欄位調整 #424
      --8
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/67
      [SubTask]docsCreation : 2. 開立工作票 #67--（reopen）
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/465
      [Improvement] let open ai get all contents before #465
      --5
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/472
      update README.md #472
      --5
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/476
      [Improvement] Prevent Sending Empty String to server #476
      --1
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/477
      [Improvement] Avoid Sending Immediately on Enter When Typing in zh-TW #477
      --8
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/95
      KM: README 撰寫 #95
      --3
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/53
      上傳 KM(Gir Flow) 到 MerMer website #53
      --5
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/495
      🧬 Connect API to database #495
      --1 主任務規劃用
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/440
      Blockchain Explorer - 013 - Get red flags data from an Address #440
      --2
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/446
      Blockchain Explorer - 019 - Get red flags data from a currency #446
      --2
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/448
      Blockchain Explorer - 021 - Get all red flags data #448
      --2
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/449
      Blockchain Explorer - 022 - Get data from a red flag #449
      --2
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/502
      BAIFA README 撰寫 #502
      --3
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/507
      fix build warning #507
      --5
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/528
      fix build warning #528
      --8

**Comment by godmmt** [comment]
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/401
      API adjustment : 005 / 006 / 009 # 401
      --8
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/356
      [MainTask] 機器人聊天室 # 356
      --1
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/412
      Backend Development # 412
      --8
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/53
      上傳 KM(Git Flow) 到 MerMer website # 53
      --5
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/54
      [MainTask] KM: 撰寫《新手指南》 # 54
      --1
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/66
      [SubTask]docsCreation : 1. 團隊例會與準備 # 66
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/67
      [SubTask]docsCreation : 2. 開立工作票 # 67
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/68
      [SubTask]docsCreation : 3. 需求規格書撰寫 # 68
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/72
      [SubTask]docsCreation : 7. 專案建立 # 72
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/73
      [SubTask]docsCreation : 8. 程式碼撰寫 # 73
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/74
      [SubTask]docsCreation : 9. 版本管理 # 74
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/75
      [SubTask]docsCreation : 10. 品質管理 # 75
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/76
      [SubTask]docsCreation : 11. 版本釋出 # 76
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/77
      [SubTask]docsCreation : 12. KM 撰寫 # 77
      --2
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/79
      [SubTask]整合所有子任務的文章＆校稿＆存放 # 79
      --3
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/423
      [Improvement] send icon, text width, delete useless data # 423
      --8
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/401
      API adjustment : 005 / 006 / 009 # 401
      --8
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/424
      API 欄位調整 # 424
      --8
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/67
      [SubTask]docsCreation : 2. 開立工作票 # 67--（reopen）
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/465
      [Improvement] let open ai get all contents before # 465
      --5
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/472
      update README.md # 472
      --5
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/476
      [Improvement] Prevent Sending Empty String to server # 476
      --1
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/477
      [Improvement] Avoid Sending Immediately on Enter When Typing in zh-TW # 477
      --8
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/95
      KM: README 撰寫 # 95
      --3
- [ ] https://github.com/CAFECA-IO/KnowledgeManagement/issues/53
      上傳 KM(Gir Flow) 到 MerMer website # 53
      --5
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/495
      🧬 Connect API to database # 495
      --1 主任務規劃用
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/440
      Blockchain Explorer - 013 - Get red flags data from an Address # 440
      --2
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/446
      Blockchain Explorer - 019 - Get red flags data from a currency # 446
      --2
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/448
      Blockchain Explorer - 021 - Get all red flags data # 448
      --2
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/449
      Blockchain Explorer - 022 - Get data from a red flag # 449
      --2
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/502
      BAIFA README 撰寫 # 502
      --3
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/507
      fix build warning # 507
      --5
- [ ] https://github.com/CAFECA-IO/BAIFA/issues/528
      fix build warning # 528
      --8
